### ISSUE TITLE
ubuntu 18.04 cbc 2.10.5 crashed with compressed file

### ISSUE THREAD

**Issue by physics1988** [coin-or/Cbc#347]
I've installed the 2.10.5 Cbc with coinbrew, but it will always crash when I run with compressed file triptim1.mps.gz.
 However with decompressed file triptim1.mps, it works well. 
[According](url) to the release log of Cbc 2.10.0, it can read compressed files.

**Comment by physics1988** [comment]
[triptim1.mps.gz](https://github.com/coin-or/Cbc/files/5521505/triptim1.mps.gz)

**Comment by tkralphs** [comment]
Indeed, it can read compressed files, but it needs to be able to link to the system library that provides that functionality. My guess is that this is not happening for some reason. You probably need to have the `libz-dev` package and and it wouldn't hurt to have `libbz2-dev`. Try installing those first and if that doesn't fix things, then attach the file `build/CoinUtils/2.11.4/config.log` here and I'll take a look.
